### PR TITLE
Add documentation for clearSelection

### PR DESCRIPTION
**Pull request by ppot**
- Provide configuration information for the clearSelection parameter in Hyperterm.js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -557,6 +557,11 @@ <h2 id="cfg"><a href="#cfg">Configuration</a></h2>
               <td>null</td>
               <td>The default width/height in pixels of a new window e.g. [540, 380]</td>
             </tr>
+            <tr>
+              <td>"clearSelection"</td>
+              <td>false</td>
+              <td>if true, selected text will automatically be copied to the clipboard</td>
```
Comment (leo) [inline index.html]: Can you make the `i` uppercase, please?

Comment (ppot) [inline index.html]: Ya sure!

**Comment by ppot** [comment]
Rebased and updated

**Comment by leo** [comment]
Thanks!

**Pull request merged**